contributed/ra_dec_dc2_run2.2i_dask.ipynb

```diff
@@ -0,0 +1,1455 @@
+    "from holoviews.plotting.util import process_cmap\n",
+    "from holoviews.streams import RangeXY"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hv.extension('bokeh')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cmap = 'viridis'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Start our Dask Cluster\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We're only going to load the RA, Dec, so we don't need that much memory for the final product.\n",
+    "There's a 42 GB limit on memory directly in the JupyterHub environment, which we will suceed in staying under.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Start a local Dask Cluster"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = Client()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This is the central root for all data\n",
+    "# You can look this up in GCRCatalogs.site_config.site_rootdir.yaml for NERSC and IN2P3\n",
+    "# As of 2021-05-28 on NERSC this is:\n",
+    "desc_data_dir = \"/global/cfs/cdirs/lsst/shared\"\n",
+    "\n",
+    "# If you have a local copy of the data you can set your own base directory here.\n",
+    "# You will need a set of the DPDD Object Table parquet files (~112 GB) for the main par\n",
+    "# And then the star truth table (1.3 GB) for the final part at the end.\n",
+    "# desc_data_dir = \"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_release = \"dr6\"\n",
+    "\n",
+    "run_data_dir = f\"DC2-prod/Run2.2i/dpdd/Run2.2i-{data_release}/object_dpdd_only\"\n",
+    "data_path = os.path.join(desc_data_dir, run_data_dir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(data_path)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "columns = ['ra', 'dec']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ddf = dd.read_parquet(data_path, columns=columns, engine='pyarrow', kwargs={'dataset': {'use_legacy_dataset': False}})"
```

Comment:
I'm not sure what is going on with the kwargs here.  Is it really necessary? Maybe explain that?

For pedagogical purposes something more like `columns=selected_columns` might be clearer.

Reply:
You're right.  It's not necessary here.  The `kwargs` dict with the `"use_legacy_dataset": False`
tells pyarrow to use the newer Arrow API, which has improvements related to filtering: it can now push filtering down to the read, both saving memory and allow for potentially faster reads if you're reading a small subset.  But we're not really using this here.